Plugins/Wox.Plugin.Sys/Main.cs

```diff
@@ -207,17 +210,6 @@ private List<Result> Commands()
                         context.API.OpenSettingDialog();
                         return true;
                     }
-                },
-                new Result
-                {
-                    Title = "Reload Plugin Data",
```

Comment:
Please reverse this, probably accidentally removed during your merge. This is from PR https://github.com/jjw24/Wox/pull/50

Reply:
done ✔
also made the subtitle translateable.

Reply:
awesome thanks!